controllers/managedfusionoffering_controller.go

```diff
@@ -81,7 +83,12 @@ func (r *ManagedFusionOfferingReconciler) Reconcile(ctx context.Context, req ctr
 	r.initReconciler(ctx, req)
 
 	if err := r.get(r.managedFusionOffering); err != nil {
-		return ctrl.Result{}, fmt.Errorf("unable to get managedFusionOffering: %v", err)
+		if errors.IsNotFound(err) {
+			r.Log.V(-1).Info("ManagedFusionOffering resource not found")
+			return ctrl.Result{}, nil
```

Comment:
How can it be that we will get into a situation where the reconcile was called but the resource does not exist?

Reply:
1. After installation is successful and ManagedFusionOffering CR exists on the cluster
2. To uninstall, we will issue a delete request on ManagedFusionOffering CR
3. After Offering is ready to be removed, we remove the finalizer from ManagedFusionOffering CR
4. This creates a delete event for ManagedFusionOffering CR and reconcile loop runs, trying to search for ManagedFusionOffering CR but it is not able to get it and throws an error.
5. Because the error is returned another reconcile gets triggered so if the ManagedFusionOffering CR is not found then we should not return error